src/features/bumpkins/components/BumpkinModal.tsx

```diff
@@ -46,133 +48,147 @@ export const BumpkinModal: React.FC<Props> = ({ onClose }) => {
   const hasSkillPoint = true;
 
   return (
-    <div className="flex flex-wrap ">
-      <img
-        src={close}
-        className="absolute w-6 top-4 right-4 cursor-pointer z-20"
-        onClick={onClose}
-      />
-      <div className="w-full sm:w-1/3 z-10 md:mr-4">
-        <div className="w-full rounded-md overflow-hidden mb-1">
-          <DynamicNFT
-            showBackground
-            bumpkinParts={state.bumpkin?.equipped as BumpkinParts}
-          />
-        </div>
-        <div>
-          <div className="mb-1 md:mb-0">
-            <OuterPanel className="relative mt-1 ">
-              <div className="flex items-center  mb-1">
-                <span className="text-xs text-shadow">Equipped</span>
+    <>
+      {showSkills ? (
```

Comment:
One small suggestion: To avoid ternaries I find it useful to do early returns. It can read a little easier

E.g.

```
if (showSkills) {
return <BumpkinSkillsModal onClose={() => setShowSkills(false)} />
}

return (
<Panel>
          <div className="flex flex-wrap">
            <img
...
```